_data/speakers.yml

```diff
@@ -27,14 +27,24 @@ sarabjot_singh:
     students in Machine Learning and to work on social problems in education, farming,
     governance and healthcare. Dr. Anand is currently adjunct faculty at IISER, Mohali.</p>
 
-# jane_doe:
-# #    keynote: false
-#     name: Jane Doe
-#     job: Artist
-#     company: Alpha
-#     photo: speaker-2
-#     social:
-#         linkedin: linkedin.url
-#         github: github.url
-#         instagram: instagram.url
-#     about: <p>The second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker.</p><p class=mb-0>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum, nemo, illo, doloribus voluptatem dolorum harum nesciunt delectus fuga qui veniam itaque reprehenderit eos error sunt rem culpa facilis! Quo, autem.</p>
+zainab_bawa:
+  keynote: true
+  name: Zainab Bawa
+  job: CEO & Co-founder
+  company: HasGeek
+  photo: Zainab.jpg
+  social:
+    twitter: https://twitter.com/zainabbawa
```

Comment:
Can you add her linkedin profile as well - https://www.linkedin.com/in/zainabbawa

Reply:
Just merged it. Adding a hotfix